cloudbuild.yaml

```diff
@@ -50,6 +50,7 @@ steps:
   dir: docker/importer
   args: ['bash', '-ex', 'run_tests.sh']
   env:
+    - CLOUD_BUILD=1
```

Comment:
Also add this to the tests in [`staging.yaml`](https://github.com/google/osv.dev/blob/c938a662b814a54f43deb71f0ab6ae682515504a/deployment/staging.yaml#L18-L19)

(more tech debt: we need to remove/consolidate that with `build-and-stage.yaml` eventually :slightly_smiling_face:)

Reply:
Like in 01b8388 ?